tests/e2e/commands/network.go

```diff
@@ -81,11 +81,30 @@ func StartNetworkWithVersion(version string) string {
 }
 
 /* #nosec G204 */
-func StopNetwork() {
+func StopNetwork(stopCmdFlags ...string) {
+	stopCmdFlasg := append([]string{
+		NetworkCmd,
+		"stop",
+		"--" + constants.SkipUpdateFlag,
+	}, stopCmdFlags...)
+	cmd := exec.Command(
+		CLIBinary,
+		stopCmdFlasg...,
+	)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Println(cmd.String())
+		fmt.Println(string(output))
+		utils.PrintStdErr(err)
+	}
+	gomega.Expect(err).Should(gomega.BeNil())
+}
+
+func GetNetworkStatus() (string, error) {
 	cmd := exec.Command(
 		CLIBinary,
 		NetworkCmd,
-		"stop",
+		"status",
```

Comment:
should we use a constant defined on cmd/ ?

Reply:
same as above - it is a good idea and i'd do it in a follow up PR